cdap-data-fabric/src/main/java/io/cdap/cdap/data2/dataset2/lib/table/leveldb/LevelDBTableCore.java

```diff
@@ -205,7 +205,7 @@ public Scanner scan(byte[] startRow, byte[] stopRow,
 
     DBIterator iterator = getDB().iterator();
     seekToStart(iterator, startRow);
-    byte[] endKey = stopRow == null ? null : createEndKey(stopRow);
+    byte[] endKey = stopRow == null ? null : createStartKey(stopRow);
```

Comment:
should we rename the `createStartKey` -> `createKey` and add javadoc to it?

Reply:
A "row" here maps to a range of rows in levelDB due to current implementation in CDAP (which is non-conventional).
Given this non-conventional implementation, I would prefer to keep the naming reflecting that by calling it createStartKey(row)

Reply:
sounds good 👍  Ideally it would be named more explicit as something like `getFirstPossibleKeyforRow` but maybe too verbose. Not a big deal. thanks for explanation!